app/models/post.rb

```diff
@@ -0,0 +1,21 @@
+class Post < ApplicationRecord
+  after_save :update_post_counter
+
+  belongs_to :user, class_name: 'User', foreign_key: 'AuthorId'
+  has_many :likes
+  has_many :comments
+
+  def most_recent_comments
+    Comment.where(PostId: id).last(5)
+  end
+
+  def updates_the_posts_counter
+    user.increment!(:PostCounter)
+  end
+
+  private :updates_the_posts_counter
+
+  validates :Title, presence: true, length: { maximum: 250 }
+  validates :CommentCounter, numericality: { greater_than_or_equal_to: 0 }
+  validates :LikeCounter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Please remove the validations, You will implement validations in the upcoming project.

Reply:
Ok , I am removing that.

Reply:
Done.